src/systemActs/ContentActs.ts

```diff
@@ -24,6 +24,7 @@ import {
     ValueClearedPayload,
     ValueRemovedPayload,
     ValueSetPayload,
+    CanFulfillIntentPayload as CanFulfillIntentPayload,
```

Comment:
CanFulfillIntentPayload will suffice ??

Reply:
Do you mean that i should change it to be just `CanFulfillIntentPayload `? Will do.

Reply:
Yes!